packages/jest/test/jest.spec.ts

```diff
@@ -34,4 +36,49 @@ describe('jest process', () => {
         expect(module.classes.root).to.equal(`test-custom__root`);
         expect(module.classes.test).to.equal(`test-custom__test`);
     });
+
+    it('should resolve default config from stylable.config.js and use the provided resolver', () => {
+        const filename = require.resolve(
+            '@stylable/jest/test/fixtures/default-config/index.st.css'
+        );
+        const content = readFileSync(filename, 'utf8');
+        const transformer = stylableTransformer.createTransformer({
+            stylable: { resolveNamespace: (ns, _srcPath) => `${ns}-custom` },
+            configPath: join(dirname(filename), 'stylable.config.js'),
+        });
+
+        const module = nodeEval(
+            transformer.process(content, filename),
+            filename
+        ) as RuntimeStylesheet;
+
+        expect(module.classes.root).to.equal(`index-custom__root green-custom__test`);
+    });
+
+    it('should use inline resolver over default config one', () => {
+        const filename = require.resolve(
+            '@stylable/jest/test/fixtures/default-config/index.st.css'
+        );
+        const content = readFileSync(filename, 'utf8');
+        const transformer = stylableTransformer.createTransformer({
+            stylable: {
+                resolveNamespace: (ns, _srcPath) => `${ns}-custom`,
+                resolveModule: createDefaultResolver(fs, {
+                    alias: {
+                        'wp-alias': join(dirname(filename), 'webpack-alias'),
+                    },
+                }),
+            },
+            configPath: join(dirname(filename), 'stylable.config.js'),
```

Comment:
I can't seem to understand the difference between the 2 resolvers, maybe have a single import and let each resolver route to a different file with a different `-st-global` on the same name class

Reply:
refactored this to use two different directories and inline `@st-namespace`